client/src/components/overview/overview.test.js

```diff
@@ -1,50 +1,749 @@
+  },
+  {
+      "thumbnail_url": "https://images.unsplash.com/photo-1488778578932-0f84d315fcae?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
+      "url": "https://images.unsplash.com/photo-1488778578932-0f84d315fcae?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=658&q=80"
+  },
+  {
+      "thumbnail_url": "https://images.unsplash.com/photo-1527431016-15eb83515018?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
+      "url": "https://images.unsplash.com/photo-1527431016-15eb83515018?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1100&q=80"
+  },
+  {
+      "thumbnail_url": "https://images.unsplash.com/photo-1534550017194-5df79ed78967?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
+      "url": "https://images.unsplash.com/photo-1534550017194-5df79ed78967?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80"
+  },
+  {
+      "thumbnail_url": "https://images.unsplash.com/photo-1525896650794-60ad4ec40d0e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
+      "url": "https://images.unsplash.com/photo-1525896650794-60ad4ec40d0e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
+  },
+  {
+      "thumbnail_url": "https://images.unsplash.com/photo-1560857792-215f9e3534ed?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
+      "url": "https://images.unsplash.com/photo-1560857792-215f9e3534ed?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80"
+  }
+];
+
+const skus = [
+  {
+      "quantity": 14,
+      "size": "7",
+      "sku_id": "1394895"
+  },
+  {
+      "quantity": 25,
+      "size": "7.5",
+      "sku_id": "1394896"
+  },
+  {
+      "quantity": 9,
+      "size": "8",
+      "sku_id": "1394897"
+  },
+  {
+      "quantity": 2,
+      "size": "8.5",
+      "sku_id": "1394898"
+  },
+  {
+      "quantity": 18,
+      "size": "9",
+      "sku_id": "1394899"
+  },
+  {
+      "quantity": 12,
+      "size": "9.5",
+      "sku_id": "1394900"
+  },
+  {
+      "quantity": 10,
+      "size": "10",
+      "sku_id": "1394901"
+  },
+  {
+      "quantity": 18,
+      "size": "10.5",
+      "sku_id": "1394902"
+  },
+  {
+      "quantity": 11,
+      "size": "11",
+      "sku_id": "1394903"
+  },
+  {
+      "quantity": 35,
+      "size": "11.5",
+      "sku_id": "1394904"
+  },
+  {
+      "quantity": 25,
+      "size": "12",
+      "sku_id": "1394905"
+  }
+]
+
+jest.mock('./overview.jsx')
+jest.mock('./addToCartButton.jsx')
+
+describe('AddToCart Component', () => {
+  it('renders the AddToCart Component', () => {
+    const addtocart = render(<AddToCart options={style}/>);
+    expect(addtocart).toBeDefined();
+  });
 });
 
-test('AddToCartButton Component should be an object', () => {
-  expect(typeof AddToCartButton).toBe('object');
+describe('AddToCartButton Component', () => {
```

Comment:
good simplistic rendering tests. They are exactly like mine currently, so i'm not judging.